src/tairzset.c

```diff
@@ -962,6 +961,138 @@ void exZrandMemberWithCountCommand(RedisModuleCtx *ctx, TairZsetObj *zobj, long
+            if (RedisModule_StringToLongLong(argv[i+1], &count) != REDISMODULE_OK) {
+                RedisModule_ReplyWithError(ctx, "ERR value is not an integer or out of range");
+                goto cleanup;
+            }
+
+            if (count < 1) {
+                RedisModule_ReplyWithError(ctx, "ERR syntax error");
+                goto cleanup;
+            }
+
+            i += 2;
+        } else if (!mstringcasecmp(argv[i], "match") && j >= 2) {
+            pat = RedisModule_StringPtrLen(argv[i+1], &patlen);
+
+            /* The pattern always matches if it is exactly "*", so it is
+             * equivalent to disabling it. */
+            use_pattern = !(pat[0] == '*' && patlen == 1);
+
+            i += 2;
+        } else {
+            RedisModule_ReplyWithError(ctx, "ERR syntax error");
+            goto cleanup;
+        }
+    }
+
+    /* Step 2: Iterate the collection. */
+    ht = zobj->dict;
+    count *= 2;
+
+    void *privdata[1];
+    /* We set the max number of iterations to ten times the specified
+     * COUNT, so if the hash table is in a pathological state (very
+     * sparsely populated) we avoid to block too much time at the cost
+     * of returning no or very few elements. */
+    long maxiterations = count*10;
+
+    /* We pass two pointers to the callback: the list to which it will
+     * add new elements, and the object containing the dictionary so that
+     * it is possible to fetch more data in a type-dependent way. */
+    privdata[0] = keys;
+    do {
+        cursor = m_dictScan(ht, cursor, dictScanCallback, NULL, privdata);
+    } while (cursor &&
+            maxiterations-- &&
+            listLength(keys) < (unsigned long)count);
+
+    /* Step 3: Filter elements. */
+    node = listFirst(keys);
+    while (node) {
+        RedisModuleString *member = listNodeValue(node);
+        nextnode = listNextNode(node);
+        int filter = 0;
+        size_t member_str_len;
+        const char * member_str = RedisModule_StringPtrLen(member, &member_str_len);
+        
+        /* Filter element if it does not match the pattern. */
+        if (!filter && 
+            use_pattern && 
+            !m_stringmatchlen(pat, patlen, member_str, member_str_len, 0)) {
+            filter = 1; 
+        }
+
+        /* Remove the element and its associted value if needed. */
+        if (filter) {
+            /* Remove both member and score */
+            m_listDelNode(keys, node);
+            node = nextnode;
+            nextnode = listNextNode(node);
+            m_listDelNode(keys, node);
+        } else {
+            /* skip the score node associated with the member node */
+            node = nextnode;
+            nextnode = listNextNode(node);
+        }
+        node = nextnode;
+    }
+
+    /* Step 4: Reply to the client. */
+    RedisModule_ReplyWithArray(ctx, 2);
+    char buf[LONG_STR_SIZE];
+    m_ll2string(buf,LONG_STR_SIZE,cursor);
```

Comment:
```suggestion
    m_ll2string(buf, LONG_STR_SIZE, cursor);
```

Reply:
You can use `.clang-format` to format your code. 
In addition, redis has used space-separated parameters since 7.0.